galaxy.yml

```diff
@@ -1,6 +1,6 @@
 namespace: community
 name: aws
-version: 3.3.0
+version: 3.4.0
 readme: README.md
```

Comment:
Is there any modules requiring tags documentations fragments (in this release) that have been included in amazon.aws 3.3.1? If so, shouldn't we increase the requirement?

Reply:
Can't find any. But increasing the doesn't hurt imo.

Reply:
I'll run the `ondemand` pipeline after trembles PR gets merged in to see if we _need_ to increment the version. afaik that should show us if anything's broken

Reply:
Any tests would currently run against stable-3, so that wouldn't catch it.  However `grep -r amazon.aws.tags plugins` shows no backported use of the tags fragment in `community.aws`.  For the sake of not adding a dependency that isn't there, let's live with 3.3.0 for now.